webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/data/CreateNodeData.scala

```diff
@@ -101,6 +99,31 @@ object Rest {
       ipAddresses:    List[String],
       timezone:       Option[NodeTimezone]
   )
+
+  object JsonCodecNodeDetails {
+
+    import com.typesafe.config.ConfigRenderOptions
+    import zio.json._
+    import zio.json.ast._
+    import zio.json.internal.Write
+
+    implicit val codecConfigValue: JsonCodec[ConfigValue] = JsonCodec(
+      new JsonEncoder[ConfigValue] {
```

Comment:
strictly speaking, we don't need codec but we are going more and more toward `zio-json`, so it makes sense to have codec for that directly I think